projects/storefrontlib/src/lib/checkout/components/shipping-address/address-form/address-form.component.spec.ts

```diff
@@ -290,14 +288,21 @@ describe('AddressFormComponent', () => {
   });
 
   it('should call countrySelected()', () => {
-    spyOn(userService, 'loadRegions').and.stub();
-
+    const spy = spyOn(userService, 'getRegions').and.returnValue(of([]));
```

Comment:
instead of assigning spy to a variable, simply call it and then refer it like this: `userService.getRegions`